### DISCUSSION TITLE
[Support]: lightning_threshold is misspelled

### DISCUSSION THREAD

**Discussion by dapowers87** [blakeblackshear/frigate#14497] [Config Support] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

The configuration field, `lightning_threshold` is misspelled. Here is ChatGPT summarizing up the differences better than I can do:

**START CHATGPT EXPLANATION**

**"Lightning"** and **"lightening"** are often confused because they sound similar, but they have very different meanings:

- **Lightning**: Refers to the natural electrical discharge in the atmosphere during a storm. It's the bright flash of light that accompanies thunder.

  *Example*: *The lightning lit up the sky during the storm.*

- **Lightening**: Refers to the process of making something lighter, either in color, weight, or intensity.

  *Example*: *She is lightening her hair before the summer.*

So, **"lightning"** is a weather phenomenon, while **"lightening"** is the act of becoming or making something lighter.

**END CHATGPT EXPLANATION**

Perhaps the solution could be to introduce a `lightening_threshold` that succeeds `lightning_threshold` in the documentation and silently allow `lightning_threshold` for backwards compatibility with a mutual exclusivity error check?

I'd be happy to be part of the solution. I feel a need to give back to this project!

Thanks for reading,
--David

P.S.
 - I'm am wishing with all of my might not to fall victim to [Muphry's Law](wikipedia.org/wiki/Muphry's_law) with this submission lol

### Steps to reproduce

This is a meta-bug

### Version

0.14.1-f4f3cfa

### In which browser(s) are you experiencing the issue with?

Brave: Version 1.71.114 Chromium: 130.0.6723.58 (Official Build) (arm64)

### Frigate config file

```yaml
# Please see https://docs.frigate.video/configuration/motion_detection/#tuning-for-large-changes-in-motion for citation
```

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

N/A

### Screenshots of the Frigate UI's System metrics pages

N/A

**Comment by hawkeye217** [comment, marked as answer]
This is incorrect. `lightning_threshold` does indeed refer to the weather phenomenon as lightning causes large changes in detected motion. 

https://docs.frigate.video/configuration/motion_detection/#tuning-for-large-changes-in-motion

The current spelling is correct, and this is not a bug.
